tests/base.py

```diff
@@ -79,6 +80,10 @@ def setUp(self):
             wing.save()
             wing_two = Wing(name="Big Apple", floor_id=floor_two.id)
             wing_two.save()
+            tag = Tag(name='Block-A',
+                        color='green',
+                        description='The description')  # noqa: E501
```

Comment:
Please avoid the use of noqa in the codebase as much as possible.

Reply:
Thanks for the feedback. Noqa has been removed and the line modified.